rasa/train.py

```diff
@@ -73,6 +72,28 @@ def train(
         config, domain, nlu_data_directory, story_directory
     )
 
+    dialogue_data_not_present = not os.listdir(story_directory)
+    nlu_data_not_present = not os.listdir(nlu_data_directory)
+
```

Comment:
this function would need some refactoring somewhen. It's getting a bit long. But probably not as part of your pr.